src/utils.js

```diff
@@ -658,23 +653,21 @@ function suggestCorrectedDateFormat(inputDate) {
 /**
  * Parses a duration string representing a past time window and converts it to milliseconds.
  *
- * @param {string} pastTimeWindow - The past time duration to parse, e.g., "20 mins", "15 minutes".
+ * @param {string} pastTimeWindow - The past time duration to parse, e.g., "20", "15".
  * @returns {number} The duration in milliseconds.
  */
 function parsePastDuration(pastTimeWindow) {
-	// Regular expression to match various formats of minute abbreviations
-	const pastDurationRegex = /^(\d+)\s*(m|mins?|minutes?)$/i;
-	const match = pastTimeWindow.match(pastDurationRegex);
+	// Check if pastTimeWindow contains non-numeric characters
+	const match = pastTimeWindow.match(/^(\d+)$/);
 
-	if (!match) {
+	const durationInMs = parseInt(pastTimeWindow) * 60 * 1000;
+
+	if (isNaN(durationInMs) || !match) {
 		throw new Error(
-			'Invalid format. The past time window should be in minutes, for example, "20 mins", "15 minutes".',
+			'Invalid format. The past time window should be integer, for example, "20", "15".',
```

Comment:
Further recommend getting a review from Jared on this message to makes sure it's consistent with others.